src/Test/Proxy/SymfonyProxy.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\Test\Proxy;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Controls the Symfony HttpCache proxy server.
+ *
+ * SYMFONY_CACHE_DIR directory to use for cache
+ *                   (default sys_get_temp_dir() + '/foshttpcache-varnish')
+ */
+class SymfonyProxy implements ProxyInterface
+{
+    /**
+     * Get Symfony cache directory
+     *
+     * @return string
+     */
+    public function getCacheDir()
+    {
+        return defined('SYMFONY_CACHE_DIR') ? SYMFONY_CACHE_DIR : sys_get_temp_dir() . '/foshttpcache-varnish';
+    }
+
+    /**
+     * Start the proxy server
+     */
+    public function start()
+    {
+        $this->clear();
+    }
+
+    /**
+     * Stop the proxy server
+     */
+    public function stop()
+    {
+        // nothing to do
+    }
+
+    /**
+     * Clear all cached content from the proxy server
+     */
+    public function clear()
+    {
+        if (is_dir($this->getCacheDir())) {
+            $fs = new Filesystem();
+            $fs->remove($this->getCacheDir());
```

Comment:
ah, i had that around because i accidentally required symfony/config which depends on the filesystem. this is the only place we use it. not sure if its worth it to have that. we could also do `system('rm -R '.$this->getCacheDir());` (and `DEL /S` for windows).

one thing i just worry here is that on misconfiguration, we could accidentally attempt to delete the whole filesystem or the home folder or cwd - this is not related to how we delete. should we sanity check the cache dir? as in: must not be empty or "/"?